Added k3S and Imapal files (#344)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/impala.md

---
name: Impala
category: Databases - Big-data
description: Apache Impala is a distributed SQL query engine designed for fast, interactive analysis of large-scale data stored in Hadoop, enabling real-time big data processing.
download_url: https://impala.apache.org/downloads.html
works_on_arm: true
supported_minimum_version:
    version_number: 4.4.0

>     release_date: 2024-08-01

Author: ```suggestion
    release_date: 2024/08/01
```

 
 
optional_info:
    homepage_url: https://impala.apache.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content: https://learn.microsoft.com/en-us/connectors/impala/
        official_docs: https://cwiki.apache.org/confluence/display/IMPALA/Building+Impala
    arm_recommended_minimum_version:
        version_number:
        release_date:
 
optional_hidden_info:
    release_notes__supported_minimum: https://issues.apache.org/jira/browse/IMPALA-12353
    release_notes__recommended_minimum:

>     other_info: According to the [documentation](https://github.com/apache/impala?tab=readme-ov-file#supported-platforms), experimental support for Arm64/Linux is there since version 4.0 but Jira ticket IMPALA-12353 confirms that the proper support is added in version 4.4.0.

Author: ```suggestion
    other_info: According to the [documentation](https://github.com/apache/impala?tab=readme-ov-file#supported-platforms), experimental support for Linux/ARM64 is introduced in version 4.0, but Jira ticket IMPALA-12353 confirms that the proper support is added in version 4.4.0.
```
